src/chapters/control-flow-and-collections/arrays.rst

```diff
@@ -0,0 +1,134 @@
+Array
+=====
+
+We learned about arrays in C# in :ref:`a previous lesson <array>`, 
+so let’s spend a moment comparing them to ``Lists``. ``Lists``  
+are generally easier to use than C#'s ``Array``. Let's see why this is.
+
+Why does C# have both ``Arrays`` and ``Lists``? The answer is
+historical, at least in part. C# is a C-style language, and arrays are
+the most basic data structure in C. Using an ``Array`` over a
+``List`` might be preferred in some circumstances, primarily for
+performance reasons (array operations are generally faster than ``List``
+operations). Also note that *Arrays are of fixed size*. You cannot
+expand or contract an ``Array`` after it is created, so you must know
+exactly how many elements it will need to hold when you create it. This
+fact is reason enough to use ``Lists`` in most scenarios.
+
+To illustrate ``Array`` usage, here is a version of the Gradebook program
+using ``Arrays`` instead of ``Lists``:
+
+.. sourcecode:: csharp
+   :lineno-start: 9
+
+   // Allow for at most 30 students
+   int maxStudents = 30;
+
+   string[] students = new string[maxStudents];
+   double[] grades = new double[maxStudents];
+
+   string input;
+   string newStudent;
+   int numStudents = 0;
+
+   Console.WriteLine("Enter your students (or ENTER to finish):");
+
+   // Get student names
+   do {
+      input = Console.ReadLine();
+      newStudent = input
+
+      if (!Equals(newStudent, "")) {
+         students[numStudents] = newStudent;
+         numStudents++;
+      }
+
+   } while(!Equals(newStudent, ""));
+
+   // Get student grades
+   for (int i = 0; i < numStudents; i++) {
+      Console.WriteLine("Grade for " + students[i] + ": ");
+      input = Console.ReadLine();
+      double grade = Double.Parse(input);
```

Comment:
can we just use double? thats what you have in the example repo

Reply:
Done!